tests/pipeline/test_pipeline_algo.py

```diff
@@ -132,6 +143,36 @@ def setUp(self):
             dtype=float,
         )
 
+        # Create a data portal holding the data in self.closes
+        data = {}
+        for sid in sids:
+            data[sid] = DataFrame({
+                "open": self.closes[sid].values,
+                "high": self.closes[sid].values,
+                "low": self.closes[sid].values,
+                "close": self.closes[sid].values,
+                "volume": self.closes[sid].values,
+                "day": [day.value for day in self.dates]
+            })
+
+        path = os.path.join(self.tempdir.path, "testdaily.bcolz")
+
+        DailyBarWriterFromDataFrames(data).write(
```

Comment:
because I was tired when I wrote it, and haven't cared enough to fix it.  Will get to it eventually.